LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2023 Hyperswitch
```

Comment:
This section specifies how to apply the Apache License to our work, and is meant to contain placeholder information itself, not the substituted values.

The bolierplate notice, if included in the source code files, should have the substituted values.

For reference, the "How to apply this license" section on https://choosealicense.com/licenses/apache-2.0/ states:

> Create a text file (typically named LICENSE or LICENSE.txt) in the root of your source code and copy the text of the license into the file. 

For licenses which need values to be substituted like MIT (https://choosealicense.com/licenses/mit/), it is explicitly stated:

> Create a text file (typically named LICENSE or LICENSE.txt) in the root of your source code and copy the text of the license into the file. Replace [year] with the current year and [fullname] with the name (or names) of the copyright holders.